### ISSUE TITLE
It is hard to parse any order funnels

### ISSUE THREAD

**Issue by neilkakkar** [PostHog/posthog#11747]
## Bug description

Any-order funnels used to look different, which clearly demonstrated that it's not a sequence funnel. But now, we lose this information.

What we see today:

<img width="575" alt="image" src="https://user-images.githubusercontent.com/7115141/189195342-a11c5683-a733-41ac-a588-0d5d922da0f0.png">

What I'd like to see: 

<img width="584" alt="image" src="https://user-images.githubusercontent.com/7115141/189195404-012e8d8a-5d4f-459f-98a8-bd6378805e5f.png">

(from our old mocks: https://www.figma.com/file/9yWtngNb1AIuf6KmXaEPJA/App-doodles?node-id=1123%3A145127 )

The overall design has changed, but the concept remains the same.

*Please describe.*  
*If this affects the front-end, screenshots would be of great help.*  

## How to reproduce

1.
2.
3.

## Environment

- [ ] PostHog Cloud
- [ ] self-hosted PostHog, version/commit: please provide

**Comment by clarkus** [comment]
I redrew any order funnels in the latest visualization style at https://www.figma.com/file/A9XRxeDm3ceikbenfTBGZl/Insights%2C-Dashboards%2C-%26-Project-Home?node-id=177%3A14274.
